### DISCUSSION TITLE
Error getting external address...

### DISCUSSION THREAD

**Discussion by CatsBy119** [GerryDazoo/Slinger#236] [Q&A]
I set this up a few weeks ago (brilliant piece of work) and configured it to connect to two Slingbox Solos. I was connecting to both boxes remotely using VLC but that stopped working (not exactly sure when). I restarted the server and it gets to a point where it says 
  Getting external IP address
  Register: Error getting external IP Address, Retrying in 10 minutes
I can still stream both Slingboxes to VLC media player on the PC where the server is installed but cannot connect from a different device even when its on the same network.
I have already configured the firewall to work with the server but turned it off anyway and have a port forwarding rule set up on my router to direct traffic to the server port to the IP address of the PC where the server is installed.
I'm assuming that the server error is causing the connection problem. 
Any ideas on how to fix this?

**Comment by CatsBy119** [comment]
Okay... issue seems to be related to using both FinderID and a static IP addresss. Gerry's notes clearly state not to use both (my bad).

**Reply by GerryDazoo** [reply]
Not your bad.

**Comment by GerryDazoo** [comment]
The root cause of the issue was the cloud service I implemented for finding your Server if you don't have a static IP or some other Dynamic DNS service failed to restart after applying security updates to the AWS instance it is running on.
This should not have affected your ability to stream video.
The AWS issue has been resolved.
